ui/package.json

```diff
@@ -20,7 +20,8 @@
     "react-dom": ">= 16.8.0",
     "react-is": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "styled-components": "^5.2.0"
+    "styled-components": "^5.2.0",
+    "tslint": "^6.1.3"
```

Comment:
lets move this to `devDependencies`, can prob do the same with the `@types` file.